frontends/PyCDE/src/pycde/system.py

```diff
@@ -101,11 +101,24 @@ def add_packaging_step(self, func: Callable):
 
   @property
   def hw_output_dir(self):
-    return self.output_directory / "hw"
+    ret = self.output_directory / "hw"
```

Comment:
nit: I am used to a style that expects function calls to be self.output_directory() to distinguish from class properties/variable names.